proxy/grpcproxy/watch.go

```diff
@@ -104,6 +105,7 @@ func (sws *serverWatchStream) recvLoop() error {
 				ch: sws.watchCh,
 
 				progress: cr.ProgressNotify,
+				filters:  v3rpc.FiltersFromRequest(cr),
```

Comment:
This is for the watcher inside the proxy. The watcher we created for etcd server is always unfiltered for better coalesce purpose. see line 116.